### ISSUE TITLE
Fields related to ticket template do not load in ticket updating page

### ISSUE THREAD

**Issue by loyolajavi** [glpi-project/glpi#16794]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0.15

### Bug description

Fields related to ticket template do not load in ticket updating:

When select a category with a template associated, the predefined fields are not loaded, mandatory fields are not highlighted, the information of the new category selected is not updated (the information balloon of the category remains empty). 
By the way, the fields are loaded when click on "Save", and of course if you dont complete a mandatory field, the glpi doesnt save the update and show a message, but it should be displayed immediately when selecting the category.

This issue only occurs in the ticket updating page, but it works fine when creating a ticket. (If you select a category with an asosciated template, the mandatory fields are highlighted, etc.)

I have these issues in the chrome browser, I don't know if they have something to do with it:
Incorrect use of <label for=FORM_ELEMENT>
The label's for attribute doesn't match any element id. This might prevent the browser from correctly autofilling the form and accessibility tools from working correctly.
To fix this issue, make sure the label's for attribute references the correct id of a form field.
14 resources

I deactivated absolutely all plugins, so that would not be causing the problem.

These bugs are happening since at least version 10.0.10 (I went from 9.5.7 to 10.0.10 directly), now i've installed 10.0.14. And I tested on 10.0.15 in a clean cloud glpi demo.

### Relevant log output

```shell
No logs related
```

### Page URL

/front/ticket.form.php

### Steps To reproduce

When change a category on an existing ticket, fails the template data loading, the predefined fields are not loaded, mandatory fields are not highlighted, the information of the new category selected is not updated (the information balloon of the category remains empty):
Only after I click on save, Glpi indicates that there are required fields and shows them highlighted.
This is a gif image:
![GlpiUpdateTicket](https://github.com/glpi-project/glpi/assets/17304522/c4cf8bc8-dac2-4de5-9b73-e9c5b6e9d87c)

This issue only occurs when modifying or updating a ticket, but it works fine when creating a ticket.

### Your GLPI setup information

Operating system: Linux 3a575a3c94d2 5.15.0-88-generic #​98-Ubuntu SMP Mon Oct 2 15:18:56 UTC 2023 x86_64
PHP 8.3.4 apache2handler (Core, FFI, PDO, Phar, Reflection, SPL, SimpleXML, Zend OPcache, apache2handler, bz2, calendar, ctype,
	curl, date, dom, exif, fileinfo, filter, ftp, gd, gettext, hash, iconv, igbinary, imap, intl, json, ldap, libxml, mbstring,
	mysqli, mysqlnd, openssl, pcre, pdo_mysql, posix, random, readline, redis, session, shmop, sockets, sodium, standard, sysvmsg,
	sysvsem, sysvshm, tokenizer, xml, xmlreader, xmlrpc, xmlwriter, xsl, zip, zlib)
Setup: max_execution_time="1200" memory_limit="2048M" post_max_size="100M" safe_mode="" session.save_handler="files"
	upload_max_filesize="100M" disable_functions="" 
Software: Apache/2.4.57 (Debian) (Apache/2.4.57 (Debian) Server at XX
)
	Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36
Server Software: mariadb.org binary distribution
	Server Version: 11.0.2-MariaDB-1:11.0.2+maria~ubu2204
	Server SQL Mode: 
	Parameters: XX@XX
	Host info: XX
	
PHP version (8.3.4) is supported.
Sessions configuration is OK.Sessions configuration is OK.
Allocated memory is sufficient.Allocated memory is sufficient.
mysqli extension is installed.mysqli extension is installed.
Following extensions are installed: dom, fileinfo, filter, libxml, json, simplexml, xmlreader, xmlwriter.Following extensions are installed: dom, fileinfo, filter, libxml, json, simplexml, xmlreader, xmlwriter.
curl extension is installed.curl extension is installed.
gd extension is installed.gd extension is installed.
intl extension is installed.intl extension is installed.
zlib extension is installed.zlib extension is installed.
The constant SODIUM_CRYPTO_AEAD_XCHACHA20POLY1305_IETF_NPUBBYTES is present.The constant SODIUM_CRYPTO_AEAD_XCHACHA20POLY1305_IETF_NPUBBYTES is present.
Database engine version (11.0.2) is supported.Database engine version (11.0.2) is supported.
No files from previous GLPI version detected.No files from previous GLPI version detected.
The log file has been created successfully.The log file has been created successfully.
Write access to /var/www/html/glpi/files/_cache has been validated. Write access to /var/www/html/glpi/files/_cron has been validated. Write access to /var/www/html/glpi/files has been validated. Write access to /var/www/html/glpi/files/_dumps has been validated. Write access to /var/www/html/glpi/files/_graphs has been validated. Write access to /var/www/html/glpi/files/_lock has been validated. Write access to /var/www/html/glpi/files/_pictures has been validated. Write access to /var/www/html/glpi/files/_plugins has been validated. Write access to /var/www/html/glpi/files/_rss has been validated. Write access to /var/www/html/glpi/files/_sessions has been validated. Write access to /var/www/html/glpi/files/_tmp has been validated. Write access to /var/www/html/glpi/files/_uploads has been validated.Write access to /var/www/html/glpi/files/_cache has been validated.
Write access to /var/www/html/glpi/files/_cron has been validated.
Write access to /var/www/html/glpi/files has been validated.
Write access to /var/www/html/glpi/files/_dumps has been validated.
Write access to /var/www/html/glpi/files/_graphs has been validated.
Write access to /var/www/html/glpi/files/_lock has been validated.
Write access to /var/www/html/glpi/files/_pictures has been validated.
Write access to /var/www/html/glpi/files/_plugins has been validated.
Write access to /var/www/html/glpi/files/_rss has been validated.
Write access to /var/www/html/glpi/files/_sessions has been validated.
Write access to /var/www/html/glpi/files/_tmp has been validated.
Write access to /var/www/html/glpi/files/_uploads has been validated.

Web server root directory configuration seems safe.Web server root directory configuration seems safe.
Sessions configuration is secured.Sessions configuration is secured.
OS and PHP are relying on 64 bits integers.OS and PHP are relying on 64 bits integers.
exif extension is installed.exif extension is installed.
ldap extension is installed.ldap extension is installed.
openssl extension is installed.openssl extension is installed.
Following extensions are installed: bz2, Phar, zip.Following extensions are installed: bz2, Phar, zip.
Zend OPcache extension is installed.Zend OPcache extension is installed.
Following extensions are installed: ctype, iconv, mbstring, sodium.Following extensions are installed: ctype, iconv, mbstring, sodium.
Write access to /var/www/html/glpi/marketplace has been validated.Write access to /var/www/html/glpi/marketplace has been validated.

**Comment by loyolajavi** [comment]
I tested it in 10.0.14 and the failure persists.
also occurs in the linked tickets:
(the ticket information balloon is empty)
![image](https://github.com/glpi-project/glpi/assets/17304522/74bf16bb-829c-4e9c-a5b2-8c55845aad7a)

The problem with this failure, for example, is that when you select a category that has mandatory fields, it does not dynamically highlight them.

**Comment by loyolajavi** [comment]
I show it with pictures to make it clear what happens:
Creation of new ticket (no failures)
![image](https://github.com/glpi-project/glpi/assets/17304522/f956a3bc-e199-4e61-b923-98702bbb5352)

Update of a ticket (), it does have the failure and does not perform the load dynamically/asynchronously and does not show the required fields, the information of the dropdown selected objects...
![image](https://github.com/glpi-project/glpi/assets/17304522/7e011395-085b-48b7-9e80-29a316ee581d)

**Comment by loyolajavi** [comment]
Update category in a ticket, fails ajax, mandatory fields are not highlighted, the data of the selected object in the dropdowns is not loaded in the "Information" icon, and other errors:
When I click on save, Glpi indicates that there are required fields and shows them highlighted.
![GlpiAjaxUpdateTicket](https://github.com/glpi-project/glpi/assets/17304522/93debbb7-94bc-4c43-9e6c-4486836c5f9d)

Create a ticekt and select the same category as in the first gif, works OK:
![GlpiAjaxNewticket](https://github.com/glpi-project/glpi/assets/17304522/2b6ba5c7-c07b-4c27-9968-f9bc02add893)

**Comment by loyolajavi** [comment]
Hello, this fails in v10.0.15 too. 
Checked in a clean Glpi in the cloud demo.

**Comment by pyero83** [comment]
Same problem here in version 10.0.15 and on the kanban view when moving a ticket.

**Comment by loyolajavi** [comment]
Hello, can I help with something to fix this issue?
